sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.2.2 (Unreleased)
+### Bug Fixes
+- Fixed the logic of `getIdentifierRawId` and `createIdentifierFromRawId` for `PhoneNumberIdentifier` to always maintain the original phone number string whether it included the leading `+` sign or not.
```

Comment:
`getIdentifierRawId` and `createIdentifierFromRawId` -> do not match the Java API